primitives/src/settlement.rs

```diff
@@ -186,8 +187,8 @@ pub enum Leg {
         sender_identity: IdentityId,
         /// The [`IdentityId`] of the receiver.
         receiver_identity: IdentityId,
-        /// The [`Ticker`] for the off-chain asset.
-        ticker: Ticker,
+        /// The [`AssetID`] for the off-chain asset.
+        asset_id: AssetID,
```

Comment:
Do we need to change the off-chain receipt to use `AssetID`?  Since they shouldn't reference an on-chain asset.  The `Ticker` field here is just informational, we could make it optional.